src/bluetooth/qbluetoothtransferreply_bluez.cpp

```diff
@@ -62,24 +61,16 @@ Q_DECLARE_LOGGING_CATEGORY(QT_BT_BLUEZ)
 
 QBluetoothTransferReplyBluez::QBluetoothTransferReplyBluez(QIODevice *input, const QBluetoothTransferRequest &request,
                                                            QBluetoothTransferManager *parent)
-:   QBluetoothTransferReply(parent), tempfile(0), source(input),
+    :   QBluetoothTransferReply(parent), objectTransfer(0), objectPush(0), client(0), tempfile(0), source(input),
     m_running(false), m_finished(false), m_size(0),
-    m_error(QBluetoothTransferReply::NoError), m_errorStr(), m_transfer_path()
+    m_error(QBluetoothTransferReply::NoError), m_errorStr()
 {
     setRequest(request);
     setManager(parent);
-    client = new OrgOpenobexClientInterface(QLatin1String("org.openobex.client"), QLatin1String("/"),
+    client = new OrgBluezObexClientInterface(QLatin1String("org.bluez.obex.client"), QLatin1String("/"),
```

Comment:
Use QStringLiteral() instead of QLatin1String() if the expression will eventually converted to a QString as it is here.

The above causes a UTF-8 to UTF-16 conversion to happen at run-time.